### PR TITLE
Update to Target Cake 1.0.0

### LINKED ISSUE #13: Add cake-addin tag to Cake.XCode NuGet package
The [NuGet Gallery](https://nuget.org) will soon include a "Cake" tab with instructions on how to use NuGet packages in Cake build scripts. I'd like to suggest that you include the tag `cake-addin` in the NuGet package of this addin, so that the NuGet Gallery can display the correct instructions to install this addin.

The UI will look similar to this:

image

Documentation on best practices on tags for Cake addins: https://cakebuild.net/docs/extending/addins/best-practices#tags

### LINKED ISSUE #15: Update Cake.XCode to target Cake v1.0.0
https://cakebuild.net/blog/2021/02/cake-v1.0.0-released

## Expected Behavior

Cake.XCode should be compatible with the current version of Cake.

## Current Behavior

Cake.XCode is not compatible with the current version of Cake, see #14 

## Possible Solution

Update Cake references to target Cake 1.0.0

## Steps to Reproduce (for bugs)
see #14

### LINKED ISSUE #16: Update Cake.XCode NuGet package icon to the new Cake-Contrib logo for Cake addins
We've recently released new icons for Cake extensions, to help more easily differentiate Cake Addins, Modules, Recipes, etc.

[cake-contrib logos](https://github.com/cake-contrib/graphics)

More information: https://github.com/cake-contrib/graphics

### PR DESCRIPTION
**Pull request by dmariogatto**
## Description
Update Cake dependencies.

## Related Issue
#13, #15 & #16

## Motivation and Context
Support for Cake 1.0.0

## How Has This Been Tested?

Existing tests have been updated and are passing. No new tests added.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: GitReleaseManager.yaml [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
       plural:  Documentation
     - name:    security
       header:  Security
       plural:  Security
+create:
+  include-footer: true
+  footer-heading: Where to get it
+  footer-content: >
+    You can download this release from
+    [nuget](https://nuget.org/packages/Cake.FileHelpers/{milestone}),
```
Comment (nils-a) [inline GitReleaseManager.yaml]:
```suggestion
    [nuget](https://nuget.org/packages/Cake.XCode/{milestone}),
```

File: GitReleaseManager.yaml [hunk 2]
```diff
@@ -1,16 +1,48 @@
...
+    or you can just reference it in a cake build script
+    with `#addin nuget:?package=Cake.FileHelpers&version={milestone}`.
```
Comment (nils-a) [inline GitReleaseManager.yaml]:
```suggestion
    with `#addin nuget:?package=Cake.XCode&version={milestone}`.
```

**Review by augustoproiete — APPROVED** [review]
LGTM!

**Comment by augustoproiete** [comment]
@dmariogatto your changes have been merged, thanks for your contribution :+1:

**Pull request merged**